components/backstage/components/backstage_list.tsx

```diff
@@ -84,6 +107,39 @@ const BackstageList = ({searchPlaceholder = localizeMessage('backstage_list.sear
         );
     }
 
+    let footer = null;
+    const page = remainingProps.page;
+    if (typeof page != 'undefined' && length) {
+        const {startCount, endCount} = getPaginationProps(length);
+        const firstPage = startCount <= 1;
```

Comment:
is this supposed to be a boolean? If so you can use the condition directly no need to assign. Also the naming is off, I recommend something like `isFirstPage` for all boolean values.